src/sinks/tcp.rs

```diff
@@ -31,11 +45,23 @@ pub enum Encoding {
     Json,
 }
 
+#[derive(Deserialize, Serialize, Debug, Default, Eq, PartialEq, Clone)]
+#[serde(rename_all = "snake_case")]
+pub struct TcpSinkTlsConfig {
```

Comment:
Since we are already name spaced under tcp we can just call this TlsConfig?

Reply:
Sure

Reply:
Actually, by that metric, shouldn't the main config just be called `Config`? I realize that's beyond the scope of this PR.

Reply:
I think it's better to have it a bit more specific than that, since searching for it is easier, while TlsConfig only exists in one place and thus is easy to search and doesn't need the long name. That's at least how I see it.

Reply:
I think concise but still greppable is a good goal for type names.